app-devel/blueprint-compiler/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=blueprint-compiler
+PKGSEC=devel
+ABSPLITDBG=0
```

Comment:
Dependencies? Also why disable this?

Reply:
> I got this error:
> 
> ```
> [WARN]:  QA (W322): Zero-byte files found:
> 
> /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler/abdist/usr/lib/python3.10/site-packages/blueprintcompiler/__init__.py
> 
> [WARN]:  QA warning(s) found, please refer to abqawarn.log ...
> [INFO]:  Packing dpkg package(s) ...
> [DEBUG]: Member 'mainline' in Architecture group 'mainline' mismatches retro.
> [INFO]:  All architecture group members 'mainline' mismatches retro: taking false branch.
> [INFO]:  Architecture loongarch64 mismatches mips32r6el: taking false branch.
> [INFO]:  Architecture loongarch64 mismatches riscv64: taking false branch.
> dpkg-deb: building package 'blueprint-compiler' in 'blueprint-compiler_0.12.0-0_loongarch64.deb'.
> In file included from /usr/bin/autobuild:13
> /usr/lib/autobuild4/proc/80-pm-pack-install.sh:19: In function `source':
> /usr/lib/autobuild4/proc/80-pm-pack-install.sh:19: error: command exited with 0
>    18 |         # A empty dir or dir with no regular files becomes a fatal error if ABSPLITDBG is set.
> >  19 |         abdie "ABSPLITDBG is set, but we can't find any symbol files in ${SYMDIR}"
>    20 |     fi
> 
> autobuild encountered an error and couldn't continue.
> ABSPLITDBG is set, but we can't find any symbol files in /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler/abdist-dbg
> ------------------------------autobuild 4.1.9------------------------------
> Go to ‘https://github.com/AOSC-Dev/autobuild4’ for more information on this error.
> [ERROR]: Build directory of the failed package: /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler
> Traceback (most recent call last):
>   File "/usr/lib/python3.10/site-packages/acbs/main.py", line 280, in build_sequential
>     invoke_autobuild(task, build_dir, scoped_stage2)
>   File "/usr/lib/python3.10/site-packages/acbs/utils.py", line 222, in invoke_autobuild
>     start_build_capture(env_dict, build_dir)
>   File "/usr/lib/python3.10/site-packages/acbs/utils.py", line 168, in start_build_capture
>     raise RuntimeError('autobuild3 did not exit successfully.')
> RuntimeError: autobuild3 did not exit successfully.
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/lib/python3.10/site-packages/acbs/main.py", line 149, in build
>     self.build_sequential(build_timings, packages)
>   File "/usr/lib/python3.10/site-packages/acbs/main.py", line 286, in build_sequential
>     raise RuntimeError(
> RuntimeError: Build directory of the failed package: /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler
> ```

Is this a "noarch" package?

Reply:
Did not know what PKGDEP it needs
and BUILDDEP is included in the BuiltKit

Reply:
> > I got this error:
> > ```
> > [WARN]:  QA (W322): Zero-byte files found:
> > 
> > /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler/abdist/usr/lib/python3.10/site-packages/blueprintcompiler/__init__.py
> > 
> > [WARN]:  QA warning(s) found, please refer to abqawarn.log ...
> > [INFO]:  Packing dpkg package(s) ...
> > [DEBUG]: Member 'mainline' in Architecture group 'mainline' mismatches retro.
> > [INFO]:  All architecture group members 'mainline' mismatches retro: taking false branch.
> > [INFO]:  Architecture loongarch64 mismatches mips32r6el: taking false branch.
> > [INFO]:  Architecture loongarch64 mismatches riscv64: taking false branch.
> > dpkg-deb: building package 'blueprint-compiler' in 'blueprint-compiler_0.12.0-0_loongarch64.deb'.
> > In file included from /usr/bin/autobuild:13
> > /usr/lib/autobuild4/proc/80-pm-pack-install.sh:19: In function `source':
> > /usr/lib/autobuild4/proc/80-pm-pack-install.sh:19: error: command exited with 0
> >    18 |         # A empty dir or dir with no regular files becomes a fatal error if ABSPLITDBG is set.
> > >  19 |         abdie "ABSPLITDBG is set, but we can't find any symbol files in ${SYMDIR}"
> >    20 |     fi
> > 
> > autobuild encountered an error and couldn't continue.
> > ABSPLITDBG is set, but we can't find any symbol files in /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler/abdist-dbg
> > ------------------------------autobuild 4.1.9------------------------------
> > Go to ‘https://github.com/AOSC-Dev/autobuild4’ for more information on this error.
> > [ERROR]: Build directory of the failed package: /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler
> > Traceback (most recent call last):
> >   File "/usr/lib/python3.10/site-packages/acbs/main.py", line 280, in build_sequential
> >     invoke_autobuild(task, build_dir, scoped_stage2)
> >   File "/usr/lib/python3.10/site-packages/acbs/utils.py", line 222, in invoke_autobuild
> >     start_build_capture(env_dict, build_dir)
> >   File "/usr/lib/python3.10/site-packages/acbs/utils.py", line 168, in start_build_capture
> >     raise RuntimeError('autobuild3 did not exit successfully.')
> > RuntimeError: autobuild3 did not exit successfully.
> > 
> > During handling of the above exception, another exception occurred:
> > 
> > Traceback (most recent call last):
> >   File "/usr/lib/python3.10/site-packages/acbs/main.py", line 149, in build
> >     self.build_sequential(build_timings, packages)
> >   File "/usr/lib/python3.10/site-packages/acbs/main.py", line 286, in build_sequential
> >     raise RuntimeError(
> > RuntimeError: Build directory of the failed package: /var/cache/acbs/build/acbs.ai_gbz5c/blueprint-compiler
> > ```
> 
> Is this a "noarch" package?

Yeah, it is.